Common/TableProducer/eventSelection.cxx

```diff
@@ -517,7 +552,7 @@ struct EventSelectionTask {
     evsel.reserve(collisions.size());
   }
 
-  void processRun2(aod::Collision const& col, BCsWithBcSelsRun2 const&, aod::Tracks const&, aod::FV0Cs const&)
+  void processRun2(aod::Collision const& col, BCsWithBcSelsRun2 const&, FullTracksIU const&, aod::FV0Cs const&)
```

Comment:
Wrong subscription to a Run 3 table.

Reply:
@vkucera Thanks, indeed, Jan Fiete also pinged me on this issue. I hope to fix it before tomorrow's tag.

Reply:
Fixed in https://github.com/AliceO2Group/O2Physics/pull/8044